### ISSUE TITLE
Add pop up message on the UI when backend fail

### ISSUE THREAD

**Issue by rizary** [haskell-CI/hackage-matrix-builder#62]
When working on this https://github.com/haskell-CI/hackage-matrix-builder/pull/58, There is a discussion on how we handle the UI when the backend fail.

@benkolera mentioned that "you should consider having a way to print the errors out somewhere on the backend failure cases. A EventWriter with an error message to some global error popup can work well."

**Comment by hvr** [comment]
I'm not sure what you mean by "error popup" but I consider "error dialog windows" rather annoying. I'd hope to have something like e.g. GitHub does when a webservice fails by showing a "banner" at the top which you can close by pressing the `x` button, e.g.

![youcantperform](https://user-images.githubusercontent.com/285533/59336751-6fb20e80-8cff-11e9-8bae-27d0d9ddfbf7.png)

**Comment by rizary** [comment]
yeah something like that. I agree that error dialog windows will be our preferable ways of showing the error. Anyway, when did you get this error?

**Comment by benkolera** [comment]
Yeah, sorry the word popup has the connotation of dialog but I didn't mean
that. Just some absolute positioned, user dismissable thing that doesn't
break control flow but at least gives some feedback that the thing the user
asked for didn't work. Way better to have a token notification area than
things just silently failing one day.

Like what is standard in bootstrap:
https://getbootstrap.com/docs/4.0/components/alerts/

**Comment by rizary** [comment]
@benkolera noted Ben, thank you.
